jalib/jalloc.cpp

```diff
@@ -36,6 +38,7 @@ static bool _initialized = false;
 JAllocArena allocArenas[MAX_ARENAS];
 static int numAllocArenas;
 
+#undef JALIB_ALLOCATOR
```

Comment:
@karya0 , I believe that the jalib allocator should be fine.  At the time, @aayushi363 was working toward a deadline (presentation on her initial port), and so she was forced to take some shortcuts.  But yes, we should now use jalloc instead of malloc.